schemas/webhooks/webhook_event.yaml

```diff
@@ -0,0 +1,31 @@
+type: object
+properties:
+  payload:
+    type: object
+    required:
+      - resource_type
+      - resource_id
+      - event
+    properties:
```

Comment:
This is missing properties that we can define as optional

Reply:
Missing optional `resource` object at the top level

Reply:
That's not needed since thanks to `disallowAdditionalPropertiesIfNotPresent: false` ([here](https://github.com/onfido/onfido-openapi-spec/blob/master/generators/common/config.yaml#L7)) we enforce the [encouraged setting](https://openapi-generator.tech/docs/generators/typescript-axios/#config-options)  to have generated code compliant with the OAS and JSON schema specifications: this allow having new fields in reply even if absent from definition and being also compliant with [Onfido Versioning policy for v3.6+](https://developers.onfido.com/guide/api-versioning-policy#how-we-version-v36) regarding non-breaking changes.

Reply:
> That's not needed since using the [encouraged setting](https://openapi-generator.tech/docs/generators/typescript-axios/#config-options) disallowAdditionalPropertiesIfNotPresent: false ([here](https://github.com/onfido/onfido-openapi-spec/blob/master/generators/common/config.yaml#L7)) to generate code compliant with the OAS and JSON schema specifications: this allows having new fields in reply even if absent from definition (compliant also with [Onfido Versioning policy for v3.6+](https://developers.onfido.com/guide/api-versioning-policy#how-we-version-v36) non-breaking changes definition).

Do you have an example of the generated class for the WebhookEvent? How will the customer be able to access the optional property if it's not defined?